src/native/corehost/bundle/extractor.cpp

```diff
@@ -103,11 +113,68 @@ void extractor_t::extract(const file_entry_t &entry, reader_t &reader)
 {
     FILE* file = create_extraction_file(entry.relative_path());
     reader.set_offset(entry.offset());
-    int64_t size = entry.size();
+    size_t size = entry.size();
```

Comment:
We use `int64_t` for this in file_entry, same for cast_size.

Reply:
Right, Thanks!